react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -178,26 +178,41 @@ class RCTAztecView: Aztec.TextView {
     }
     
     // MARK: - Paste handling
-    
-    private func html(from pasteboard: UIPasteboard) -> String? {
-        guard let data = pasteboard.data(forPasteboardType: kUTTypeHTML as String) else {
-            return nil
+    private func read(from pasteboard: UIPasteboard, uti: String, documentType: DocumentType) -> String? {
+        guard let data = pasteboard.data(forPasteboardType: uti as String),
```

Comment:
Small detail:
`uti` is already `String` so this extra `as String` is not needed.

Maybe we can change `uti`'s type to be `CFString` to make it simpler on the caller side?